pupil-spa/src/app/familiarisation-colour/familiarisation-colour.component.ts

```diff
@@ -1,15 +1,37 @@
 import { Component } from '@angular/core';
 import { Router } from '@angular/router';
+import {
+  AccessArrangementsConfig,
+  AccessArrangements,
+  accessArrangementsDataKey
+} from '../access-arrangements';
+import { StorageService } from '../services/storage/storage.service';
 
 @Component({
   selector: 'app-familiarisation-colour',
   templateUrl: './familiarisation-colour.component.html',
   styleUrls: ['./familiarisation-colour.component.scss']
 })
 export class FamiliarisationColourComponent {
-  constructor(private router: Router) {}
+  selectedContrast;
+  contrastSettings;
+
+  constructor(
+    private router: Router,
+    private storageService: StorageService
+  ) {
+    this.contrastSettings = AccessArrangementsConfig.contrastSettings;
+  }
+
+  selectionChange(selectedContrast) {
+    this.selectedContrast = selectedContrast;
+  }
 
   onClick() {
-    this.router.navigate(['sign-in-success']);
+    const accessArrangements = new AccessArrangements;
+    accessArrangements.contrast = this.selectedContrast || 'bow';
+    this.storageService.setItem(accessArrangementsDataKey, accessArrangements);
+
+    this.router.navigate(['access-settings']);
```

Comment:
@danwilliams I would expect the navigation to differ for the two following scenarios:
- User is choosing colour contrast pref after login
- User is editing colour contrast pref from access settings page